CONTRIBUTING.md

```diff
@@ -75,18 +75,19 @@ Following is described how to set up the integration for local development.
             "ParameterKey": "HumioIngestToken", 
             "ParameterValue": "YOUR-SECRET-INGEST-TOKEN" 
         },
-        { 
-            "ParameterKey": "HumioCloudWatchLogsAutoSubscription", 
-            "ParameterValue": "true" 
-        },
         { 
             "ParameterKey": "HumioCloudWatchLogsSubscriptionPrefix", 
             "ParameterValue": "humio" 
+        },
+        {
+            "ParameterKey": "Version",
+            "ParameterValue": "v0.0.0"
```

Comment:
Should this always be v.0.0.0 or should it be updated to current version?

Reply:
Ah, just read the bellow, maybe have the value be something like [your-version-here] instead of v.0.0.0 as that looks kinda legiatamate.